TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -538,7 +596,17 @@ public static void ScanTokensLineContinuationGroup(IList<TokensLine> continuatio
                     multiStringConcatBitPosition.Set(pos, true);
                 }
             }
-            Scanner.ScanTokensLine(virtualContinuationTokensLine, initialScanState, compilerOptions, copyTextNameVariations, multiStringConcatBitPosition);
+            BitArray continuationBitPositions = null;
+            if (continuationPositions.Count > 0)
+            {
+                continuationBitPositions = new BitArray(concatenatedLine.Length);
+                foreach (int pos in continuationPositions)
+                {
+                    continuationBitPositions.Set(pos, true);
+                }
+            }
+            Scanner.ScanTokensLine(virtualContinuationTokensLine, initialScanState, compilerOptions, copyTextNameVariations,
+                new ContinuationPositions(continuationBitPositions, multiStringConcatBitPosition));
```

Comment:
Why using a BitArray instead of the original `List<int>` ?

Reply:
To make access be in O(1) rather than in O(n) where n is the size of the list..

Reply:
Ok, use an `HashSet<int>` then. And this will remove the need to create and set the BitArray.

Reply:
hum, we need the length of the BitArray, and the length is the final Concatenated line length, here the only goal is to have faster access during the scanning of the concatenated line, not during the construction of the Concatenated line.